src/helpers/dom/element.js

```diff
@@ -643,7 +649,7 @@ export function getScrollableElement(element) {
       computedOverflowY = computedStyle.getPropertyValue('overflow-y');
       computedOverflowX = computedStyle.getPropertyValue('overflow-x');
 
-      if (computedOverflow === 'scroll' || computedOverflowX === 'scroll' || computedOverflowY === 'scroll') {
+      if (props.indexOf(computedOverflow) !== -1 || props.indexOf(computedOverflowX) !== -1 || props.indexOf(computedOverflowY) !== -1) {
```

Comment:
Maybe [Array.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) will be yet more clear?